src/Wallabag/ImportBundle/Consumer/AbstractConsumer.php

```diff
@@ -52,6 +52,13 @@ protected function handleMessage($body)
 
         $this->import->setUser($user);
 
+        if (false === $this->import->validateEntry($storedEntry)) {
+            $this->logger->warning('Entry is invalid', ['entry' => $storedEntry]);
```

Comment:
It would be nice to have only the URL(s) here instead of an array.

This would require every import having a `getEntryUrl()` method that returns the appropriate value for each appropriate key (`uri`/`href` and so on).

Other than that, :+1:

Reply:
The `validateEntry` method only checks for empty url. Which means in case the url is empty I'll put an empty string in the error message :thinking:

Reply:
Maybe we could identify unique data for each import implementation for the logging part?
e.g. `guid` for firefox bookmarks, `meta` for pinboard export and title otherwise.

Reply:
`$storedEntry` isn't the Entry entity, it's an array of the imported json data which means it should contains too much data (like the content of the entry for example)

Reply:
That's why I was not fond of passing the whole array to the logger. ;)

Reply:
(I guess @j0k3r forgot the word "not" :D)

Reply:
It should **not** contains :)